local-test-infra/local-pod-under-test.yaml

```diff
@@ -24,6 +24,9 @@ spec:
         - command: [ "/bin/bash", "-c", "echo 'logs' && tail -f /dev/null" ]
           image: quay.io/testnetworkfunction/cnf-test-partner:latest
           name: test
+          ports:
+            - name: "testport"
+              containerPort: 8080
```

Comment:
Should we add a `type: TCP` on this as well?

Reply:
It's not a mandatory field. Also, when omitted, it defaults to TCP. As per the API doc  https://kubernetes.io/docs/reference/kubernetes-api/workload-resources/pod-v1/#Container
```
Protocol for port. Must be UDP, TCP, or SCTP. Defaults to "TCP".
```

Reply:
Sounds good.  Didn't know if we wanted to be explicit of the type or not.